verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -202,6 +202,18 @@ paths:
         schema:
           type: string
           pattern: '^[a-f\d]{24}$'
+      - name: parse_start_date
+        in: query
+        description: The date on which to begin parsing/ingesting data.
```

Comment:
those dates are "included" I believe, better be explicit here and/or in the openapi specs about it.

Reply:
Good call, done.